dnf-plugins-core.spec

```diff
@@ -9,7 +10,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
+Epoch:          1
```

Comment:
Unfortunatelu some extras has provide with epoch 1 therefore I have to overwrite it with same epoch and higher version.